mubench.pipeline/tests/tasks/implementations/test_collect_versions.py

```diff
@@ -10,7 +12,7 @@ def test_finds_all_versions(self):
         project = create_project("-project-")
         v1 = create_version("-v1-", project=project)
         v2 = create_version("-v2-", project=project)
-        uut = CollectVersionsTask()
+        uut = CollectVersionsTask(True)
```

Comment:
I would prefer the tests to simulate production mode, instead of development mode.